js/js_test.go

```diff
@@ -571,3 +571,20 @@ func TestUint8Array(t *testing.T) {
 		t.Errorf("Non-empty byte array is not externalized as a Uint8Array")
 	}
 }
+
+func TestInternalizeZeroVal(t *testing.T) {
+	type S struct {
+		*js.Object
+
+		string `js:"string"`
```

Comment:
> but in order that we can cover all Go types I think we need to follow the struct field tags approach...

Are methods like `String`, `Float`, `Bool` `Uint64`, etc., not sufficient? What's not possible without struct field tags?